src/paradigma/gait_analysis.py

```diff
@@ -379,6 +381,35 @@ def quantify_arm_swing(path_to_feature_input: Union[str, Path], path_to_predicti
     df_aggregates['segment_duration_ms'] = df_aggregates['segment_duration_s'] * 1000
```

Comment:
Is this necessary for your code?

Reply:
This was already present in the _main_ branch. However, I think the reason why this is useful (especially in _tsdf_) is because the _time_ column can now be stored as integer, instead of float.